contrib/epee/include/net/net_parse_helpers.h

```diff
@@ -128,11 +128,51 @@ namespace net_utils
     return true;
   }
 
+  inline
+    bool parse_url_ipv6(const std::string url_str, http::url_content& content)
```

Comment:
That function makes me a bit uncomfortable, but I think it's fine to leave as is. I might come back later and see if it can be removed.